app/models/setting/remote_execution.rb

```diff
@@ -1,88 +0,0 @@
-class Setting::RemoteExecution < Setting
-
-  ::Setting::BLANK_ATTRS.concat %w{remote_execution_ssh_password remote_execution_ssh_key_passphrase remote_execution_sudo_password remote_execution_effective_user_password remote_execution_cockpit_url remote_execution_form_job_template}
```

Comment:
oh, we can't do without this... This can be:

1. hacked into the BLANK_ATTRS
2. wait for the validations to get in and set through the `allow_empty`

Reply:
I don't know if I understood that correctly. Should I include `BLANK_ATTRS` in the `engine.rb` file?

Reply:
Yeah, that would be an option, but we are planning to remove that validation completely, so it's better to wait for https://github.com/theforeman/foreman/pull/8470

Reply:
Support for validation is now merged in core, would be nice to use it here.